Mage.Sets/src/mage/cards/n/NearbyPlanet.java

```diff
@@ -26,13 +31,20 @@ public NearbyPlanet(UUID ownerId, CardSetInfo setInfo) {
         // Rangeling
         this.addAbility(new SimpleStaticAbility(Zone.ALL, new NearbyPlanetEffect()));
 
+        this.addAbility(new WhiteManaAbility());
```

Comment:
That’s wrong. Must be AnyColor ability instead. See code and comments example from OmoQueenOfVesuva.

Also must have “dependencies” code from OmoQueenOfVesuva too.